src/cudamatrix/cu-matrix.cc

```diff
@@ -2012,30 +2040,116 @@ void CuMatrixBase<Real>::AddCols(const CuMatrixBase<Real> &src,
 }
```

Comment:
You can submit a pull request for this issue.

On Wed, Aug 19, 2015 at 6:50 PM, Daniel Povey <dpovey@gmail.com> wrote:

> Oh, yes, you're right, there should be no AddCols that accepts a
> std::vector.
> Dan
>
>
> On Wed, Aug 19, 2015 at 6:48 PM, Xingyu Na <notifications@github.com>
> wrote:
>
>> In src/cudamatrix/cu-matrix.cc
>> <https://github.com/kaldi-asr/kaldi/pull/83#discussion_r37488643>:
>>
>> > @@ -2012,30 +2040,116 @@ void CuMatrixBase<Real>::AddCols(const CuMatrixBase<Real> &src,
>> >  }
>>
>> I mean maybe the version of AddCols that take std::vector as index list
>> is not necessary any more. Because I notice that the version of CopyCols
>> that take std::vector as index list is deleted.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/kaldi-asr/kaldi/pull/83/files#r37488643>.
>>
>
>